lib/test/t_HMatrix_LDLt.cxx

```diff
@@ -58,24 +58,17 @@ int main(int, char *[])
     ResourceMap::SetAsUnsignedInteger("HMatrix-MaxLeafSize", 10);
 
     HMatrixFactory hmatrixFactory;
-    if (!hmatrixFactory.IsAvailable()) throw NotYetImplementedException(HERE);
+    if (!PlatformInfo::HasFeature("hmat"))
+      throw NotYetImplementedException(HERE);
```

Comment:
that's probably not useful anymore because hmat tests are enabled conditionally in cmake

Reply:
ok